test/ruby/monitoring/monitoring.rb

```diff
@@ -0,0 +1,174 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Integration tests for vkit generated client library for
+# Stackdriver Monitoring APIs.
+#
+require "google/cloud/monitoring/v3"
+
+if ARGV.length != 1
+  puts "Usage: #{$0} <project_id>"
+  exit
+end
+
+PROJECT_NAME = ARGV[0].freeze
+
+#
+# Integration test for GroupServiceApi.create_group
+#
+GroupServiceApi = Google::Cloud::Monitoring::V3::GroupServiceApi
+group_service_api = GroupServiceApi.new
+Group = Google::Monitoring::V3::Group
+group = Group.new(
+  display_name: "displayName",
+  filter: "resource.type = gce_instance")
+
+formatted_project_name = GroupServiceApi.project_path(PROJECT_NAME)
+created_group = group_service_api.create_group(formatted_project_name, group)
+puts "New group created: #{created_group.name}"
+
+#
+# Integration test for GroupServiceApi.update_group
+#
+group_to_update = Group.new(
+  name: created_group.name,
+  display_name: "displayNameUpdate",
+  filter: "resource.type = gce_instance"
+)
+updated_group = group_service_api.update_group(group_to_update)
+puts "Group updated: #{updated_group.display_name}"
+
+#
+# Integration test for GroupServiceApi.get_group
+#
+group_id = created_group.name.split('/')[-1]
```

Comment:
`GroupServiceApi.match_group_from_group_name created_group.name` might be better?